src/lib/analytics.js

```diff
@@ -69,7 +69,14 @@ function sendToGoogleAnalytics({
     metric_value: value,
     // This should already by set globally, but to ensure it's consistent
     // with the web-vitals library, set it again.
-    navigation_type: navigationType,
+    // Override for 'navigational-prefetch' for the prefetch origin trial
+    // experiment (https://github.com/GoogleChrome/web.dev/pull/9532)
+    navigation_type:
+      navigationType === 'navigate' &&
+      performance.getEntriesByType('navigation')[0].deliveryType ===
```

Comment:
Do you want to also handle the cases of browsers that don't support the performance timeline API, like getNavigationType does?

(It could probably be done more concisely than it is there with the nullish coalescing syntax, assuming this goes through a build step to elide that for older UAs.)

Reply:
I did think about that but it's loaded as a module and every browser that supports that _should_ support this?

But added it anyway.